ExampleMod/Content/Items/Accessories/ExampleShield.cs

```diff
@@ -12,6 +13,7 @@ public class ExampleShield : ModItem
 	{
 		public override void SetStaticDefaults() {
 			Tooltip.SetDefault("This is a modded shield.");
+			CreativeItemSacrificesCatalog.Instance.SacrificeCountNeededByItemId[item.type] = 1;
```

Comment:
What's the default?

Reply:
Also use `Type` instead of `item.type` in this and 40 other places.

Reply:
What do you mean "whats the default?" There is no default, so you have to set it. I wanted to keep the option of people not wanting their items to be researchable, which you can easily do by just not setting it. Doesn't crash or anything. You set a static variable and it makes sense in load times, so I just put it there. Theoretically you can put it anywhere, I guess.